src/openassetio-python/tests/package/managerApi/test_managerinterface.py

```diff
@@ -71,6 +71,15 @@ def test_when_not_overridden_then_returns_empty_dict(self):
         assert info == {}
 
 
+class Test_ManagerInterface_flushCaches:
+    def test_method_defined_in_cpp(self, method_introspector):
+        assert not method_introspector.is_defined_in_python(ManagerInterface.flushCaches)
+        assert method_introspector.is_implemented_once(ManagerInterface, "flushCaches")
+
+    def test_is_noop(self, a_host_session):
+        assert ManagerInterface().flushCaches(a_host_session) is None
```

Comment:
[fix] Similarly, is this testing anything? I think the name of this test at least needs to change, as it would still pass even if `flushCaches` here wasn't a noop, since it always returns void (and thus evaluates to `None`)

Reply:
It was just a way that I could still use the word assert in the test case, rather than just calling the function. You're right though, we otherwise don't test void returns. I'll get rid of it.

Reply:
Updated as part of d0c780cb